libs/custom-paper-handler/src/driver/driver.ts

```diff
@@ -110,13 +123,12 @@ export const REAL_TIME_ENDPOINT_IN = 3;
 export const REAL_TIME_ENDPOINT_OUT = 4;
 export const PACKET_SIZE = 65536;
 
+// The number of times transferInGeneric will retry after receiving unexpected data.
+const MAX_TRANSFER_IN_ATTEMPTS = 20;
```

Comment:
We've never seen the bad byte sequence twice in a row so 20 feels like a sufficiently high limit.